variables.tf

```diff
@@ -363,6 +390,22 @@ Each object in the map consists of the following properties:
 DESCRIPTION
 }
 
+variable "linked_service_cosmosdb_mongoapi" {
+  type = map(object({
+    name              = string
+    connection_string = optional(string)
+    database          = optional(string)
```

Comment:
This resource has more arguments like `server_version_is_32_or_higher`, `desccription`, `integration_runtime_name`, `parameters`, `annotations`, `additional_properties`, don't we need them?

Reply:
This PR mainly supports Data Factory Dataset Cosmosdb Mongoapi. As for linked_service_cosmosdb_mongoapi, it is just a dependent resource. According to the original PR from Gerry, we just need to support some basic properties.